clients/include/testing_gbmv.hpp

```diff
@@ -167,22 +113,45 @@ void testing_gbmv(const Arguments& arg)
     // argument sanity check before allocating invalid memory
     if(M < 0 || N < 0 || lda < KL + KU + 1 || !incx || !incy || KL < 0 || KU < 0)
     {
```

Comment:
See the symm hemm which sets an invalid_size bool then you can just invalid_size || !M || !N and use the bool again for status check.